launches load on start up

sxv2721 opened the pull request:
used component did mount to load launches from SpaceX's founding May 6th 2002 to today using Moment

fixes #6

sxv2721 committed:
launches load on start up

used component did mount to load launches from SpaceX's founding May 6th 2002 to today using Moment

bursteways commented on src/components/DateInput/index.js:
```diff
@@ -21,6 +21,9 @@ export class DateInput extends React.Component {
             end: e.target.value
         });
     }
+    componentDidMount = () => {
```
The native lifecycle methods from React use a different syntax and should not use `arrow functions`. The reason is because whenever you define an arrow function, it creates a reference to `this` and it is bound to the class level method. Because it's not really reusable to the component, we don't need to keep a reference to it.

```
componentDidMount() {
  ...
}
```

sxv2721 replied:
I replaced the arrow function with regular function syntax

sxv2721 committed:
fixed DateInput componentDidMount

changed componentDidMount in DateInput to not use an arrow function.

bursteways reviewed (approved):
🚀

bursteways commented on src/components/DateInput/index.js:
```diff
@@ -21,6 +21,9 @@ export class DateInput extends React.Component {
+    componentDidMount(){
+        this.props.getLaunches(this.state.start, this.state.end);
+    }
     render = () => {
```
Render should be the same

sxv2721 committed:
converting arrow funcs

converted all arrow functions of react lifecycle methods into regular function syntax, and removed unecessary lifecycle methods
